concepts/05 UI Components/Accordion/00 Getting Started with Accordion/00 Getting Started with Accordion.md

```diff
@@ -0,0 +1,9 @@
+#include tutorials-intro-installationnote
+
+The Accordion UI component contains several panels displayed one under another.
+
+This tutorial shows how to add an Accordion to the page and configure its core features. As a result, you will create the following UI component:
```

Comment:
> This tutorial shows how to add an Accordion to **the page** and configure **its** core features.

A potential issue in the sentence. The pronoun "it" points at "the page" rather than "an Accordion" because, "the page" stands last and has a definite article. How to rewrite? For example:

```suggestion
This tutorial shows how to add an Accordion to the page and configure the component's core settings. As a result, you will create the following UI component:
```